src/main.cpp

```diff
@@ -116,7 +117,7 @@ void usage(std::ostream& out)
   out << "    --include FILE add an #include file before preprocessing" << std::endl;
   out << "    -l [search|filename]" << std::endl;
   out << "                   list kernel probes or probes in a program" << std::endl;
-  out << "    -p PID         filter actions and enable USDT probes on PID" << std::endl;
+  out << "    -p, --pid PID  filter actions and enable USDT probes on PID" << std::endl;
```

Comment:
If this is a pure cleanup, did you want to add `--command`, `--quiet`, `--verbose`, `--list`, `--debug`, `--emit-warnings`, etc.?

Reply:
Yeah, why not, I'll add those too.

Reply:
Since you're doing it all up, I also noticed that `--test-mode` is undocumented for example. Did we want to ensure that everything is actually documented?